components/Policies/PolicyPage.tsx

```diff
@@ -62,6 +49,42 @@ export default function PolicyPage({
       </Stack>
 
       <PolicyContent policy={policy} />
+
+      <div className={style.sharedValues}>
+        <p className={style.subHeading}>Our Shared Values</p>
+        <hr className={style.bottomBorder}></hr>
+        <p className={style.text1}>
+          How we interact with each other determines what we can accomplish
+        </p>
+        <p className={style.text2}>On this website, we ask you to act with:</p>
```

Comment:
Should we make these strings translatable?

Reply:
Yeah that is a good idea.

Reply:
Just out of curiosity, is this an expectation that developers should always provide translations for their code moving forward?

Reply:
I think keeping new code "translatable" is a good goal, however I don't think devs need to provide any language other than English in the initial PRs